lib/core/network/api_service.dart

```diff
@@ -396,7 +396,13 @@ class APIService {
       '/givtservice/v1/PaymentProvider/checkoutsession/$guid/parent-control-validation',
     );
 
-    final response = await client.get(url);
+    final response = await client.get(
```

Comment:
do we need to add that problem+json thing?

Reply:
Endpoint was returning 500 "No MediaTypeFormatter is available to read an object of type 'Object' from content with media type 'application/problem+json'." so I thought this might fix it, but it works with out it now --- no, i guess we don't _need it_